config/locales/en/coronavirus_landing_page.yml

```diff
@@ -3,6 +3,17 @@ en:
     meta_title: "Coronavirus (COVID-19): what you need to do"
     meta_description: "Find out about the government response to coronavirus (COVID-19) and what you need to do."
     page_header: "Coronavirus (COVID-19): what you need to do"
+    stay_at_home:
+      pretext: "Stay at home"
```

Comment:
Are quote marks necessary in this yml? Seems like it works elsewhere without. Another thing to look at tidying later...

Reply:
It was required before as we had `"Stay at home:"` but not anymore